habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -258,7 +258,7 @@ def create_sim_config(
 
         sensor_specifications = []
         for sensor in _sensor_suite.sensors.values():
-            sim_sensor_cfg = habitat_sim.SensorSpec()
+            sim_sensor_cfg = habitat_sim.CameraSensorSpec()
```

Comment:
Hi @Skylion007 ! I noticed that in hab-lab sensors we don't have SensorSubTypes, and was wondering what the reasoning for that was - would it be a good idea to add this enum?

For context, I check that SensorSubTypes are correct before initializing a CameraSensorSpec() here: 
https://github.com/facebookresearch/habitat-sim/pull/1090/files#diff-1c82f32ca8e8fe4738a140dd923ac12f5d482300e28446788e41eafa248a62a1R66-R103

Reply:
I'm also not sure how BumpSensor is initialized, since this isn't something I was able to find in hab-sim or hab-lab. I know we can't initialize it with CameraSensorSpec as we do the other sensors though. Any insight to this would be appreciated! @Skylion007 @bigbike @mathfac @erikwijmans @abhiskk

Reply:
We don't actually have implementations of BumpSensor, I don't believe.

Reply:
Yes, we have only [CollisionMeasure](https://github.com/facebookresearch/habitat-lab/blob/aa24551fd9942b11b7af4a076d9c05e080594d08/habitat/tasks/nav/nav.py#L670), but that would be great to have BumpSensor, but it looks like it can be HabitatLab Task sensor, not Simulator one.